enricher/api/src/main/java/io/fabric8/maven/enricher/api/AbstractHealthCheckEnricher.java

```diff
@@ -30,25 +32,67 @@ public AbstractHealthCheckEnricher(EnricherContext buildContext, String name) {
         super(buildContext, name);
     }
 
+    // Available configuration keys
+    protected enum Config implements Configs.Key {
+
+        probeMode {{ d = "all";}};
```

Comment:
@nicolaferraro i agree with you on the first/last/all, how does something like this 
```
<config>
  <probes>
       <container-name-1/>
       <container-name-2/>
 </probes>
</config> 
```
when the probes is empty then we assume "all" and add probes to all containers